src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/Routes.scala

```diff
@@ -22,18 +22,19 @@ object Routes {
     *
     * @param resources the resources operations
     */
-  def apply(resources: Resources[Task])(implicit cache: DistributedCache[Task],
+  def apply(resources: Resources[Task])(implicit cache: CacheAggregator[Task],
                                         indexers: Clients[Task],
                                         store: FileStore[Task, AkkaIn, AkkaOut],
                                         aclsOps: AclsOps,
                                         config: AppConfig): Route = {
     import indexers._
-    implicit val um = marshallers.sparqlQueryUnmarshaller
+    implicit val um                                          = marshallers.sparqlQueryUnmarshaller
+    implicit val CacheAggregator(projectCache, viewCache, _) = cache
```

Comment:
I would break that out because you're also importing the resolver cache in the scope.